### PR TITLE
updated translation tables

### PR DESCRIPTION
**Pull request by sara-rn**
additional translation tables from https://en.wikibooks.org/wiki/Algorithm_Implementation/Miscellaneous/Base64#C_2 and 09BF850BE5DA44A1C3629A1F62813A83

### CODE REVIEW

File: data-manipulation/encoding/base64/decode-data-using-base64-via-dword-translation-table.yml [hunk 1]
```diff
@@ -12,6 +13,7 @@ rule:
       - Data::Encode Data::Base64 [C0026.001]
     examples:
       - 9efa86b43b4367bcdc1591aee59bda25:0x10001000
+      - 09BF850BE5DA44A1C3629A1F62813A83:0x10001100
```
Comment (williballenthin) [inline data-manipulation/encoding/base64/decode-data-using-base64-via-dword-translation-table.yml:16]: please lowercase this for consistency

**Review by williballenthin — APPROVED** [review]
see one online comment

**Comment by williballenthin** [comment]
@sara-rn would you please register your GH handle on the google side so the CLA check passes?

**Comment by sara-rn** [comment]
fixed hash, format and GH account

**Review by williballenthin — APPROVED** [review]
thank you!

**Pull request merged**